ores/metrics_collectors/util.py

```diff
@@ -1,6 +1,4 @@
 
 
 def format_set(s):
-    l = list(s)
-    l.sort()
-    return "{" + ", ".join(repr(value) for value in l) + "}"
+    return "{" + ", ".join(repr(value) for value in sorted(s)) + "}"
```

Comment:
~~Isn't this just, json.dumps(sorted(s))?  Or does the formatting have to be more specific than that?~~

Oh, weird brace language.  NVM.

Reply:
Right.  Should be deterministic.  Formatting JSON is not deterministic.